pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+DJANGO_SETTINGS_MODULE=oneanddone.settings
+addopts = --reuse-db
+sensitive_url=mozilla\.org
```

Comment:
I've moved the contents of `pytest.ini` into `setup.cfg`, and I agree that we should enable `flake8` tests on Travis. Running it locally I see there are a few issues, so I will make that a separate issue to enable those on Travis which can also address all the flake8 failures.